<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="spectrogram_plot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>600</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="temporal_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="params_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Sweep parameters</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Beginning freq.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ending freq.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="begfreq_textedit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="endfreq_textedit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dur_textedit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="srate_textedit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling freq.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savesweep_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Saving location</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="gensweep_button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>50</y>
      <width>270</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playsweep_button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>120</y>
      <width>270</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Play sweep</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
